internal/cli/service.go

```diff
@@ -733,6 +740,21 @@ func removeDuplicateStrings(list []string) []string {
 	return slices.Compact(list)
 }
 
+func findSnippets(fileNames []string) ([]string, []string) {
```

Comment:
Minor nit: this is a rare case where I'd probably use named return values to make it a little more clear what this is returning.

Reply:
done